types/src/lib.rs

```diff
@@ -68,3 +68,26 @@ pub struct SendTransactionRequest {
 pub enum SendTransactionError {
     MalformedTransaction,
 }
+
+#[derive(CandidType, Debug, Deserialize, PartialEq)]
+pub enum UpdateTransactionError {
+    UnmodifiableTransaction,
+    UnknownTxId,
+    OneTransactionUpdateAlreadyConfirmed,
+    InsufficentFees,
+}
+
+// Structure necessary to keep track of previous sent transactions fees
```

Comment:
Isn't `TransactionWithFees` only used internally in the library?

Reply:
Indeed I talked to Islam and he told me that there is no way currently to run the bitcoin-developer-preview with the latest ic/rs/bitcoin/ version from the GitLab.

So I think that the best way for the moment is to:
- restrain this file to types accessible from the Bitcoin library that is: Satoshi, OutPoint, Utxo, UpdateTransactionError, Address and UtxosDelta.
- and move all these types to the library

I will do that.